include/deal.II/lac/block_vector_base.h

```diff
@@ -2264,6 +2262,17 @@ void BlockVectorBase<VectorType>::extract_subvector_to (ForwardIterator
     }
 }
 
+
+template <typename VectorType>
+inline
+bool BlockVectorBase<VectorType>::supports_distributed_data() const
+{
+  // Create a dummy vector of type VectorType to check
+  // supports_distributed_data even if BlockVectorBase contains no block
+  VectorType dummy;
+  return dummy.supports_distributed_data();
```

Comment:
@bangerth For example have a look at ```include/deal.II/la_parallel_vector.templates.h``` quite a number of ```operator```s have a signature like this:
```
925     template <typename Number>                                                   
926     Vector<Number> &                                                             
927     Vector<Number>::operator += (const VectorSpaceVector<Number> &vv)            
```